modules/margin/src/main/java/com/opengamma/sdk/margin/CcpsResult.java

```diff
@@ -39,6 +39,23 @@
   private final List<String> ccpNames;
 
   //-------------------------------------------------------------------------
+  /**
+   * Gets the list of available CCPs, may be empty.
+   * 
+   * @return the list of available CCPs
+   */
+  public List<Ccp> getCcps() {
+    List<Ccp> ccps = new ArrayList<>();
+    for (String name : ccpNames) {
+      try {
+        ccps.add(Ccp.of(name));
+      } catch (RuntimeException ex) {
+        // ignore, server may be newer than client
```

Comment:
Should we log a suggestion that the developer should check for a newer version of the SDK?

Reply:
The SDK has no logging. I don't want to add some just for this.